lib/packets/resultset_header.js

```diff
@@ -18,6 +18,15 @@ function ResultSetHeader (packet, bigNumberStrings, encoding)
     this.infileName = packet.readString(undefined, encoding);
   }
 
+  // issue#288: Add changedRow if exists, reference to https://github.com/sidorares/node-mysql2/issues/288
+  if (encoding) {
+    var message = packet.readString(undefined, encoding);
+    var m = message.match(/\schanged:\s*(\d+)/i);
```

Comment:
@sushantdhiman According to [https://dev.mysql.com/doc/internals/en/packet-OK_Packet.html]，the`EOF` is `human readable status information`。In other words, the `EOF` is not in one good format.